src/main/java/com/extendedclip/papi/expansion/player/PlayerExpansion.java

```diff
@@ -181,6 +181,8 @@ public String onRequest(OfflinePlayer player, String identifier) {
                 return "now available in the server expansion";
             case "displayname":
                 return p.getDisplayName();
+            case "player_list_name":
```

Comment:
I personally suggest using `list_name` instead of `player_list_name` as it would just sound extremely odd.

Reply:
I agree. Done.